ocaml/networkd/bin/network_server.ml

```diff
@@ -53,7 +53,9 @@ let on_shutdown signal =
   let dbg = "shutdown" in
   Debug.with_thread_associated dbg
     (fun () ->
-      debug "xcp-networkd caught signal %d; performing cleanup actions." signal ;
+      debug "xcp-networkd caught signal %s (%d); performing cleanup actions."
+        Fmt.(to_to_string Dump.signal signal)
```

Comment:
That's nice... will it print things like `SIGTERM`?

Reply:
yes! waiting for a host to reboot to test this out outside of `utop`

Reply:
I was working on a fix before the bug got opened... https://github.com/xapi-project/xen-api/pull/6228